Fix GNOME -> CF in trademark policy (#175)
commonhaus/foundation: policies/trademark-policy.md

    - Correct: [CF Mark]‘s technical conferences are held ...

- Do Not Use CF Marks that are not listed, or use Forms of CF Marks that vary from the list. Even a seemingly innocuous deviation may create a new and different trademark.
    - Do not shorten, abbreviate or create acronyms out of CF Trademarks.

>     - A trademark that is depicted as or more words should never be compressed into one word or vise versa.

Author: ```suggestion
    - A trademark that is depicted as or more words should never be compressed into one word or vice versa.
```

I don't know Latin, but in English I've always seen this spelled with a "c".

    - Correct: [CF Mark]‘s technical conferences are held ...

- Do Not Use CF Marks that are not listed, or use Forms of CF Marks that vary from the list. Even a seemingly innocuous deviation may create a new and different trademark.
    - Do not shorten, abbreviate or create acronyms out of CF Trademarks.

>     - A trademark that is depicted as or more words should never be compressed into one word or vice versa.

Author: ```suggestion
    - A multiword trademark should never be compressed into one word or vice versa.
```

This sentence didn't make sense.
